refactor: refactor aws accounts basic/advanced (#19)
threeport/user-docs: docs/aws/advanced-aws-setup.md

# Advanced AWS Setup

> Threeport can be configured to manage resources in an AWS account that is seperate from the one it is currently deployed in.

Reviewer: ```suggestion
Use this documentation to configure Threeport to manage resources in an AWS account that is separate from the account Threeport is deployed in.
```

> When wanting to manage a kubernetes runtime and subsequently, a workload in a external AWS account, use the following way to setup AWS cross account access:

Reviewer: ```suggestion
If using the same AWS account for Threeport and the workloads it will manage, follow the directions in the [Basic AWS Setup guide](../aws/basic-aws-setup.md)
```
